docs/contributing/style-guide.md

```diff
@@ -0,0 +1,420 @@
+
+```
+# Not recommended
+Netdata can monitor RAM, disk I/O, MySQL queries per second and lm-sensors.
+
+# Recommended
+Netdata can monitor RAM, disk I/O, MySQL queries per second, and lm-sensors.
+```
+
+
+## Markdown syntax
+
+The Netdata documentation uses the Markdown syntax for styling and formatting. If you're not familiar with how it works, please read the [Markdown introduction post](https://daringfireball.net/projects/markdown/) by its creator, followed by [Mastering Markdown](https://guides.github.com/features/mastering-markdown/) guide from GitHub.
+
+We also leverage the power of the [Material theme for MkDocs](https://squidfunk.github.io/mkdocs-material/), which features several [extensions](https://squidfunk.github.io/mkdocs-material/extensions/admonition/), such as the ability to create notes, warnings, and collapsible blocks.
+
+You can follow the syntax specified in the above resources for the majority of documents, but the following sections specify a few particular use cases.
+
+### References to UI elements
+
+If you need to instruct your reader to click a user interface (UI) element inside of a Netdata interface, you should reference the label text of the link/button with Markdown's (`**bold text**`) tag.
+
+```markdown
+Click on the **Sign in** button.
+```
+
+!!! note
+    
+    Avoid using directional language to orient readers, because not every reader can use instructions like "look at the top-left corner" to find their way around an interface.
+
+
+### Language-specific syntax highlighting in code blocks
+
+Our documentation uses the [Highlight extension](https://facelessuser.github.io/pymdown-extensions/extensions/highlight/) for syntax highlighting. Highlight is fully compatible with [Pygments](http://pygments.org/), allowing you to highlight the syntax within code blocks in a number of interesting ways.
+
+For a full list of languages, see [Pygment's supported languages](http://pygments.org/languages/). Netdata documentation will use the following for the most part: `c`, `python`, `js`, `shell`, `markdown`, `bash`, `css`, `html`, and `go`. If no language is specified, the Highlight extension doesn't apply syntax highlighting.
+
+Include the language directly after the three backticks (`` ``` ``) that start the code block. For highlighting C code, for example:
+
+````
+```c
+inline char *health_stock_config_dir(void) {
+    char buffer[FILENAME_MAX + 1];
+    snprintfz(buffer, FILENAME_MAX, "%s/health.d", netdata_configured_stock_config_dir);
+    return config_get(CONFIG_SECTION_HEALTH, "stock health configuration directory", buffer);
+}
+```
+````
+
+And the prettified result:
+
+```c
+inline char *health_stock_config_dir(void) {
+    char buffer[FILENAME_MAX + 1];
+    snprintfz(buffer, FILENAME_MAX, "%s/health.d", netdata_configured_stock_config_dir);
+    return config_get(CONFIG_SECTION_HEALTH, "stock health configuration directory", buffer);
+}
+```
+
+### Line numbers in code blocks
```

Comment:
This is a bit much and I don't see a difference in GitHub between the two versions. Not to mention that line numbers can easily change. I would get rid of the whole section.